<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox_v1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S12</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Tanque1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>162</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanque 1</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Tanque2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanque 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S21</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s22">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S22</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_b1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
